packages/docs-components/src/knobs/defaultComponents.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import { KnobComponentProps, KnobComponents } from './types'
+
+const KnobField: React.FunctionComponent<KnobComponentProps> = props => (
+  <div
+    style={{
+      borderBottom: '1px solid rgb(234, 234, 234)',
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+      padding: '3px 0',
+    }}
+  >
+    {props.children}
+  </div>
+)
+
+const KnobControl: React.FunctionComponent<KnobComponentProps> = props => (
+  <span style={{ verticalAlign: 'middle', width: '150px' }}>{props.children}</span>
+)
+
+const KnobLabel: React.FunctionComponent<KnobComponentProps> = props => (
+  <span style={{ marginRight: 5 }}>{props.content || <code>{props.name}</code>}</span>
+)
+
+const KnobBoolean: React.FunctionComponent<KnobComponentProps> = props => (
+  <input
+    onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+      props.setValue(e.target.checked)
+    }}
+    type="checkbox"
+    value={props.value}
+  />
+)
+
+const KnobSelect: React.FunctionComponent<KnobComponentProps> = props => (
+  <select
+    onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
+      props.setValue(e.target.value)
+    }}
+    value={props.value}
+  >
+    {props.values &&
+      props.values.map(option => (
+        <option key={option} value={option}>
```

Comment:
Have you considered using <Dropdown>?

Reply:
These components are part of `docs-component` package which should be consumable even without Stardust.

I am going to add mapping with our components in a separate PR.